rest-client-reactive-quickstart/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
```

Comment:
Should probably be removed too.

It has reference on Quarkus versions.... which would is updated regularly.

Reply:
removed